charts/vh-bookings-api/values.yaml

```diff
@@ -16,26 +16,28 @@ java:
       resourceGroup: vh-infra-core-{{ .Values.global.environment }}
       usePodIdentity: true
       secrets:
-        - servicebusqueue--connectionstring
-        - connectionstrings--vhbookings
         - applicationinsights--instrumentationkey
+        - azuread--tenantid
+        - connectionstrings--vhbookings
+        - servicebusqueue--connectionstring
     vh-bookings-api:
       excludeEnvironmentSuffix: false
       resourceGroup: vh-infra-core-{{ .Values.global.environment }}
       usePodIdentity: true
       secrets:
-        - azuread--clientsecret
         - azuread--clientid
-        - KinlyConfiguration--SipAddressStem
-        - FeatureFlags--StaffMemberFeature
-        - FeatureFlags--EJudFeature
+        - azuread--clientsecret
+        - kinlyconfiguration--sipaddressstem
+        - launchdarkly--sdkkey
```

Comment:
does the secret not have a dash in it like schedular jobs?
https://github.com/hmcts/vh-scheduler-jobs/blob/a742904962c03d0b7428fd25cf1459db1b9c85a9/charts/vh-scheduler-jobs/values.yaml#L45

Reply:
nope